lib/std/json/static.zig

```diff
@@ -205,7 +218,7 @@ fn internalParse(
             return try std.fmt.parseFloat(T, slice);
         },
         .Int, .ComptimeInt => {
-            const token = try source.nextAllocMax(allocator, .alloc_if_needed, options.max_value_len.?);
+            const token = try source.nextAllocMax(allocator, options.allocate, options.max_value_len.?);
```

Comment:
Similar to above, the allocation is freed.

Reply:
Addressed